pages/signup.tsx

```diff
@@ -2,6 +2,36 @@ import { FC, useState } from "react";
 import { SignupForm } from "@components/SignupForm";
 import { useAuth } from "@auth/Auth";
 import { MagicLinkModal } from "@components/MagicLinkModal";
+import { Alert } from "@components/Alert";
+
+const shutdownLevel = parseInt(
+  `${process.env.NEXT_PUBLIC_SHUTDOWN_LEVEL || 0}`
+);
+
+const preShutdownTexts = {
+  title: `Registrierung deaktiviert`,
+  message: (
+    <>
+      Stadtpuls wird am <strong>31. Januar 2023</strong> eingestellt.
+      <br />
+      Es ist deshalb nicht mehr möglich, sich zu registrieren.
```

Comment:
This is completely optional, but this new block is almost exactly the same as in the signin component, isn't it? Perhaps these things could be extracted and only the part of the text that changes ("einzuloggen" / "registrieren") provided as a parameter or something like that.

But honestly, since this is about shutdown, we can also decide to ignore this.

Reply:
That was exactly my thought. Why optimize something the shutdown. I didn't bother clean it up. If you're fine with it, I'll let it as is.